### PR TITLE
Flex grid homework

### PR DESCRIPTION
**Pull request by dragan-andrei**
# Flex&Grid. I can say that this Homework About all knolegde that we have about HTML&CSS.

https://github.com/dragan-andrei/Tekwill_FE_gr11/assets/161748972/df91b67b-5abe-4b4b-bb5d-5ed400d02326

### Выкладываю, можно сказать *заранее*. Может, если будет возможность, то вы сможете мне что-нибудь подсказать, чтобы до 9 числа работа была готова 😄

### CODE REVIEW

**Review by CozlovschiNichita — APPROVED** [review]
it's hard

File: cats.css [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+    height: auto;
+    object-fit: cover;
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 10px;
+}
+
+
+.woff {
+    position: relative;
```
Comment (IvanCeban) [inline cats.css:85]: Try ```position: absolute; display: flex, and align-items:center; justify-content: center```
In this case you will resolve issue with centering the content.

File: cats.css [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+    background-color: rgba(216, 214, 214, 0.9);
+    color: #201f1f;
+    width: 100%;
+    justify-content: center;
+    height: 100%;
+    transition: opacity 0.3s linear;
+    align-content: center;
+}
+
+.rightsidebar div.dog1:hover .woff {
```
Comment (IvanCeban) [inline cats.css]: No need to repeat opacity props. 
You can set it just once:
```
.rightsidebar>div:hover .woff {
    opacity: 1;
}```

File: cats.css [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+    width: 100%;
+    height: auto;
+}
+
+.mainpole .cat1{
+    position: relative;
+    grid-row: 1 / span 2;
+    grid-column: 1 / span 2;
+    background-image: url("cats/1.jpeg");
+    background-position: center;
```
Comment (IvanCeban) [inline cats.css]: dont repeat yourself. 
for example ```background-position and background-size``` are the same for all cats

File: cats.css [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+    background-size: cover;
+
+}
+
+.meo {
+    display: flex;
+    opacity: 0;
+    position: relative;
+    top: 94%;
```
Comment (IvanCeban) [inline cats.css:188]: better use position absolute for "meo". Will be more flexible positioning

File: cats.css [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+    align-content: flex-end;
+
+}
+
+.mainpole .cat1:hover .meo {
+    opacity: 1;
+    top:94%;
+
+}
+.mainpole .cat2:hover .meo {
```
Comment (IvanCeban) [inline cats.css]: Don't repeat yourself.

**Review by IvanCeban — CHANGES_REQUESTED** [review]
You did great job!
The page looks nice and responsiveness is great!
Just cleanup a bit your css (follow DRY principle - Don't repeat yourself)

**Comment by dragan-andrei** [comment]
> You did great job! The page looks nice and responsiveness is great! Just cleanup a bit your css (follow DRY principle - Don't repeat yourself)

Thank you! Now my code looks much better. It became more readable and it became clearer to me what was going on.

**Review by IvanCebanPVT — APPROVED** [review]
Great job!

**Review by IvanCeban — APPROVED** [review]
Great Job!

**Pull request merged**